yjit/src/codegen.rs

```diff
@@ -114,10 +114,13 @@ pub struct JITState<'a> {
 
     /// Stack of symbol names for --yjit-perf
     perf_stack: Vec<String>,
+
+    /// When true, this block is the first block compiled by gen_block_series().
+    first_block: bool,
```

Comment:
Maybe we could make this an unsigned integer and then we could know how long block series can get (could be interesting)

Reply:
We already have the max number of `get_num_versions(blockid, ctx)` in YJIT stats, but I guess you want to have the max count of all versions for a `BlockId` regardless of `inline` and `reg_mapping`?